config/foreman-proxy-content.yaml

```diff
@@ -20,6 +20,7 @@
   - foreman_proxy::plugin::pulp
   - foreman_proxy_content
   - puppet
+  - apache::mod::status
```

Comment:
is this entry even needed? :)

Reply:
The compile order in Puppet can matter. For this particular one, probably not since it just includes apache if needed:
https://github.com/puppetlabs/puppetlabs-apache/blob/c97699bf72efe4d86b6aec7ba2b665ee41b36803/manifests/mod/status.pp#L37

Reply:
compilation seems to be happy without, yay